mlperf_logging/result_summarizer/result_summarizer.py

```diff
@@ -452,7 +452,7 @@ def _check_and_update_system_specs(desc_keys, column_name, query=None):
         # Construct prefix portion of the row.
         try:
             _check_and_update_system_specs('division', 'division')
-            system_specs['availability'] = None
+            _check_and_update_system_specs('status', 'availability')
```

Comment:
Did you test this change? There is no `availability` in the system json files (e.g., `https://github.com/mlcommons/training_results_v1.0/blob/master/Google/systems/tpu-v4-2048-TF.json`)

I put it there mainly because the old result summarizer also displays `None` in `availability` for every benchmark, but I'm not sure why `availability` even needs to be displayed in the first place.